core/src/integration/java/com/radixdlt/integration/targeted/rev2/mempool/REv2MempoolFillAndEmptyTest.java

```diff
@@ -140,7 +140,7 @@ private void fillAndEmptyMempool(DeterministicTest test) {
     var mempoolDispatcher =
         test.getInstance(0, Key.get(new TypeLiteral<EventDispatcher<MempoolAdd>>() {}));
 
-    while (mempoolReader.getCount() < 1000) {
+    while (mempoolReader.getCount() < 100) {
```

Comment:
This is possibly a less useful test now - could we maybe explicitly set the mempool larger? Happy for you to do this in a follow up PR 👍